projects/components/form/src/form.component.ts

```diff
@@ -390,4 +413,18 @@ export class PsFormComponent implements OnChanges, OnInit, OnDestroy {
     }
     this._savebar.cd.markForCheck();
   }
+
+  private createErrorCardObserver() {
+    if (!this._errorCardObserver) {
+      const options = {
+        root: null as any, // relative to document viewport
+        rootMargin: '-100px', // margin around root. Values are similar to css property. Unitless values not allowed
+        threshold: 1.0, // visible amount of item shown in relation to root
+      };

```

Comment:
I think we should add a delay here to the options

Reply:
It seems like the typescript implementation of the IntersectionObserver doesn't have such options yet. [see here](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.intersectionobserver.html)